scripts/build/Platform/Windows/deploy_cdk_applications.cmd

```diff
@@ -39,6 +39,8 @@ FOR /F "tokens=4 delims=:" %%a IN ("%ASSUME_ROLE_ARN%") DO SET O3DE_AWS_DEPLOY_A
 
 IF "%O3DE_AWS_PROJECT_NAME%"=="" (
     SET O3DE_AWS_PROJECT_NAME=%BRANCH_NAME%-%PIPELINE_NAME%-Windows
+    SET slashreplace=
+    call SET O3DE_AWS_PROJECT_NAME=%%O3DE_AWS_PROJECT_NAME:/=%slashreplace%%%
 )

```

Comment:
Curious: Can we just use "SET O3DE_AWS_PROJECT_NAME=%%O3DE_AWS_PROJECT_NAME:/=%slashreplace%%%" here? I think the call keyword is for calling another batch?

Reply:
My understanding is you need the `call` here to execute [both layers](https://stackoverflow.com/questions/2772456/string-replacement-in-batch-file) of variable expansion, since we're doing it in 2 levels:
{O3DE_AWS_PROJECT_NAME:/={slashreplace}}

BTW, during testing I also tried just `SET O3DE_AWS_PROJECT_NAME=%O3DE_AWS_PROJECT_NAME:/=-%`, but it resulted in the previous content of `O3DE_AWS_PROJECT_NAME` being blown away, even though that syntax works as desired on the cmd line. So, seems there's some discrepancy in how the env vars are being interpolated between the cmd line vs. in the batch script. The format in the commit is what worked most predictably via script execution.